data_steward/constants/validation/participants/writers.py

```diff
@@ -1,7 +1,16 @@
+from constants.bq_utils import (WRITE_APPEND, WRITE_TRUNCATE)
+from constants.validation.participants.identity_match import (
+    MATCH, MISMATCH, MISSING,
+    PERSON_ID_FIELD, FIRST_NAME_FIELD, MIDDLE_NAME_FIELD, LAST_NAME_FIELD,
+    EMAIL_FIELD, PHONE_NUMBER_FIELD, SEX_FIELD, ZIP_CODE_FIELD, STATE_FIELD,
+    CITY_FIELD, ADDRESS_ONE_FIELD, ADDRESS_TWO_FIELD, BIRTH_DATE_FIELD, VALIDATION_FIELDS)
+
+WRITE_APPEND = WRITE_APPEND
```

Comment:
All of these assignments can be removed

Reply:
Yes, but static analysis tools complain when you import something and don't use it.  And I can see the point.  How do you know if something should actually be imported if it isn't used in the module?  The tools don't look outside of the module to see how the constants are used elsewhere and therefore don't know that they are used elsewhere.  This is a quirk that makes sense to me.

Reply:
I see. I would consider this a bug in the analysis tools. For Codacy, we can ignore issues for unused imports here.